### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by danvaida**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| aws\_account\_id\_hub | AWS account number containing the TGW hub | `string` | n/a | yes |
+| aws\_login\_profile | Name of the AWS login profile as seen under ~/.aws/config used for assuming cross-account roles | `any` | n/a | yes |
+| ram\_resource\_association\_id | Identifier of the Resource Access Manager Resource Association | `string` | n/a | yes |
+| role\_to\_assume\_hub | IAM role name to assume in the AWS account containing the TGW hub (eg. ASSUME-ENG-OPS) | `string` | n/a | yes |
+| aws\_account\_id\_satellite | AWS account number containing the TGW satellite | `string` | `""` | no |
+| destination\_cidr\_block | CIDR to be routed | `string` | `""` | no |
+| role\_to\_assume\_satellite | IAM role name to assume in the AWS account containing the TGW satellite (eg. ASSUME-ENG-OPS) | `string` | `""` | no |
```
Comment (snovikov) [inline README.md]: To make it clean let's rename it to `ASSUME-ROLE-SATELLITE`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+| aws.satellite | n/a |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| aws\_account\_id\_hub | AWS account number containing the TGW hub | `string` | n/a | yes |
+| aws\_login\_profile | Name of the AWS login profile as seen under ~/.aws/config used for assuming cross-account roles | `any` | n/a | yes |
+| ram\_resource\_association\_id | Identifier of the Resource Access Manager Resource Association | `string` | n/a | yes |
+| role\_to\_assume\_hub | IAM role name to assume in the AWS account containing the TGW hub (eg. ASSUME-ENG-OPS) | `string` | n/a | yes |
```
Comment (snovikov) [inline README.md]: To make it clean let's rename it to `ASSUME-ROLE-HUB`

**Review by snovikov — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (danvaida) [reply README.md]: Thanks for the reminder! I remember seeing that later on but then forgot to correct it.

File: README.md (hunk 2 above)
Comment (danvaida) [reply README.md]: Thanks for the reminder! I remember seeing that later on but then forgot to correct it.

**Review by cytopia — COMMENTED** [review]
This looks pretty good to me. As Travis CI integration is currently not working, can you either switch to GitHub actions or manually check and report findings of:

```
  - make lint
  - make gen
  - git diff --quiet || { echo "Build Changes"; git diff; git status; false; }
```

**Review by snovikov — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+You can read more about how Terraform handles this [here][5].
+
+Obviously, all the [supported authentication][6] methods can also be used.
+
+### Routing
+
+For creating the network routes on the satellite side, the module expects to
+find the keyword "_private_" in the name of the subnets, so that it may collect
+their IDs and those of their associated routing tables.
```
Comment (snovikov) [inline README.md]:
```
$ make lint
...
./README.md:42:their IDs and those of their associated routing tables. 
[ERR] Files with trailing spaces found.
```

**Review by snovikov — CHANGES_REQUESTED** [review]

**Review by snovikov — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<!-- Add bullet points for how to test this PR -->
+<!-- * What is the desired goal one can test again -->
+<!-- * How to test against the desired goal -->
+<!-- * Optionally add dry-run instructions -->
+
+
+## How to roll out
+<!-- Add information for how to roll out this PR -->
+<!-- * How to roll out (cli or CI/CD tools) -->
+<!-- * Where to roll out (affected environments) -->
```
Comment (danvaida) [inline .github/pull_request_template.md:17]: This line is irrelevant in the context of this being a module and also an open-source one.

File: .github/workflows/terraform.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  test:
+    runs-on: ubuntu-latest
+    name: "Terraform test"
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+
+      - name: "make test"
+        run: |
+          make test ARGS="-var assumer_account_role_name=test -var assumer_account_id=1234567890"
```
Comment (danvaida) [inline .github/workflows/terraform.yml]: These vars are bogus.

Comment (snovikov) [reply .github/workflows/terraform.yml]: it s a copy from Travis file

File: .github/pull_request_template.md (hunk 4 above)
Comment (snovikov) [reply .github/pull_request_template.md:17]: agreement for PRs are project dependent and nothing to do opensource. I think we can do it as we want

File: .github/workflows/terraform.yml (hunk 5 above)
Comment (snovikov) [reply .github/workflows/terraform.yml]: removed

**Review by snovikov — APPROVED** [review]

**Pull request merged**